#11917 - Careful access of annotations on class objects/fix incorrect display of inherited members (#11919)
sphinx-doc/sphinx: CHANGES.rst

  Patch by Colin Marquardt.
* #11598: Do not use query components in URLs for assets in EPUB rendering.
  Patch by David Runge.

> * #11917: Fixes some corner-cases involving inherited class members on python 3.9
>   and earlier. Patch by Janet Carson.

Reviewer: ```suggestion
* #11917: Fix rendering of annotated inherited members for Python 3.9.
  Patch by Janet Carson.
```

We do not support 3.8 and below so there is no need to mention it.

Author: added
